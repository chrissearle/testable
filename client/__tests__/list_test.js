import React from "react";
import renderer from 'react-test-renderer';
import List from "../list";

const tweets = [
  {
    id: 123,
    name: "@TrendingGithub",
    owner: "facebook",
    repo: "jest",
    languages: {
      "JavaScript": 1233
    }
  }
];

describe("List", () => {
  test("should render a list of tweets", () => {
    const list = renderer.create(
      <List tweets={tweets} />
    ).toJSON();

    expect(list).toMatchSnapshot();
  })
});
