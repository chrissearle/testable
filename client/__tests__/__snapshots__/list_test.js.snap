// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List should render a list of tweets 1`] = `
<ul>
  <li>
    <span>
      <a
        href="github.com/facebook/jest"
      >
        facebook
        /
        jest
      </a>
      : 
    </span>
    <span>
      <span>
        JavaScript
        , 
      </span>
    </span>
  </li>
</ul>
`;
